tools/refract-filter.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+
+# This is just a
```

Comment:
What do you mean by taken from somewhere? I have written it to aid my variable property name efforts so this is the upstream.